auth-web/src/components/auth/staff/account-management/StaffPendingAccountsTable.vue

```diff
@@ -200,12 +201,16 @@ export default class StaffPendingAccountsTable extends Mixins(PaginationMixin) {
 
   private formatDate = CommonUtils.formatDisplayDate
 
-  @Watch('searchParams', { deep: true })
+  @Watch('searchParams', { deep: true, immediate: true })
   async searchChanged (value: TaskFilterParams) {
     this.searchParamsExist = this.doSearchParametersExist(value)
-    this.tableDataOptions = { ...this.getAndPruneCachedPageInfo(), page: 1 }
+    const itemsPerPage = this.getNumberOfItemsFromSessionStorage() || this.DEFAULT_ITEMS_PER_PAGE
+    if (this.cachedPageInfo()) {
```

Comment:
Caching only exists when navigating outside the page and coming back (gets pruned)